source/omni_order_api.rst

```diff
@@ -0,0 +1,1224 @@
+| Requires authentication: Yes
+
+A. Request
+----------
+
+This API endpoint should used by POS terminals to fetch incoming orders from OMNI. This API is lightweight and suitable to be used for polling for new orders.
+
+This endpoint will only return said orders exactly **once**.
+
+It will only return said orders only **once**.
+
+The store is derived from the store token string.
+
+HTTP Header
+___________
+
+=================== =========== =======================
+Parameter           Required    Description
+=================== =========== =======================
+Content-Type        Yes         application/json
+Authorization       Yes         store token string
+=================== =========== =======================
+
+.. code-block::
+
+    Content-Type: application/json
+    Authorization: token example-store-token
+
+Example of API call request using cURL (JSON). To avoid HTTP 100 Continue, please specify "Expect:" as a header.
+
+.. code-block:: bash
+
+    $ curl -X GET -H "Content-Type: application/json" -H "Authorization: token example-store-token" -H "Expect:" https://host.com/api/store/orders/sync
+    
+
+B. Response
+-----------
+
+In response to these API calls, Omni will reply with the following data in JSON:
+
+=================== ==================
+Variable            Description
+=================== ==================
+orders              An array of order objects
+=================== ==================
+
+Up to 100 new orders will be returned, each order object has the following `fields`_.
+
+The order objects will only be returned **once**.
+
+If there are no new orders, it returns an empty array :code:`{ "orders" : [] }`
+
+
+Here are examples of API responses:
+
+
+If call to sync order API is successful (JSON):
+
+.. code-block:: bash
+
+    HTTP/1.0 200 OK
+    Vary: Accept
+    Content-Type: application/json
+    Allow: GET, OPTIONS
+    [Redacted Header]
+
+    {
+        "orders": [
+            {
+                [Redacted Content]
+            },
+            {
+                [Redacted Content]
+            },
+            {
+                "id": 1,
+                "number": "FR9TL74P",
+                "total": 36364.0,
+                "tax": 3636.0,
+                "serviceCharge": 0.0,
+                "grandTotal": 41000.0,
+                "tableNumber": "A1",
+                "channel": 2,
+                "notes": "No lettuce",
+                "status": 1,
+                "deliveryStatus": 10,
+                "statusText": "New",
+                "storeName": "store test",
+                "userID": 1,
+                "created": 1564045835,
+                "paymentMethod": 1,
+                "paymentStatus": 1,
+                "user": {
+                    "id": 1,
+                    "name": "test",
+                    "phone": "+628111111111"
+                },
+                "items": [],
+                "delivery_info": null
+            }
+        ]
+    }
+
+    
+..  Get Order
```

Comment:
Is this supposed to be commented? (not sure if the syntax coloring is off in github,  or is this supposed to be commented)

Reply:
Yes, that is supposed to be commented, Pak selwin said that they should use sync order instead

Reply:
got it